packages/plugin-evm/src/actions/transfer.ts

```diff
@@ -57,11 +69,51 @@ export const transferAction = {
         runtime: IAgentRuntime,
         message: Memory,
         state: State,
-        options: any
+        options: any,
+        callback?: HandlerCallback
     ) => {
+        console.log("Transfer action handler called");
         const walletProvider = new WalletProvider(runtime);
         const action = new TransferAction(walletProvider);
-        return action.transfer(runtime, options);
```

Comment:
why is options ignored now? Prefer an LLM options over explicitly passed ones?

Reply:
I found that the previous way of directly passing options into action.transfer was in

```typescript
async transfer(
runtime: IAgentRuntime,
params: TransferParams
): Promise<Transaction>
```
The parameters cannot be parsed out smoothly, so I refer to the writing method of plugin-solana

https://github.com/ai16z/eliza/blob/main/packages/plugin-solana/src/actions/transfer.ts#L116

Manually built parameters replace those passed to action.transfer.